streams/test-utils/src/test/java/org/apache/kafka/streams/TopologyTestDriverTest.java

```diff
@@ -416,9 +418,16 @@ private Topology setupTopologyWithInternalTopic(final String firstTableName,
         return builder.build(config);
     }
 
+    private Properties defaultProperties() {
+        final Properties properties = new Properties();
+        properties.put(StreamsConfig.DEFAULT_KEY_SERDE_CLASS_CONFIG, Serdes.ByteArraySerde.class);
+        properties.put(StreamsConfig.DEFAULT_VALUE_SERDE_CLASS_CONFIG, Serdes.ByteArraySerde.class);
+        return properties;
+    }
+
```

Comment:
Sorry, I realized it now. Why do you not just pass `config` where you now pass `defaultProperties()`? In this way we would not need `defaultProperties()` anymore.

Reply:
Good catch, Thank you!